### PR TITLE
[CoreLogic.pm - processPartySkillUse] Timeout absence fix.

### PR DESCRIPTION
**Pull request by Mortimal**
Fixed timeout problem.

### CODE REVIEW

**Comment by Mortimal** [comment]
> 
> 
> I think this is not the best solution. I have a 1 line solution.

show me

File: src/AI/CoreLogic.pm [hunk 1]
```diff
@@ -2692,6 +2692,7 @@ sub processPartySkillUse {
 		for (my $i = 0; exists $config{"partySkill_$i"}; $i++) {
 			next if (!$config{"partySkill_$i"});
 			next unless checkSelfCondition("partySkill_$i");
+			$ai_v{"partySkill_$i"."_time"} = time;
```
Comment (c4c1n6kr3m1) [inline src/AI/CoreLogic.pm]: is this the right  place? what will happen  if succes through checkSelfCondition but fail on checkPlayerCondition ?

File: src/AI/CoreLogic.pm [hunk 2]
```diff
@@ -2802,6 +2804,7 @@ sub processPartySkillUse {
 				$party_skill{isSelfSkill} ? $party_skill{owner} : $party_skill{targetActor},
 				$party_skill{prefix},
 			);
+			$timeout{"partySkill_".$party_skill{skillTimeoutId}."_timeout"}{time} = time;
```
Comment (c4c1n6kr3m1) [inline src/AI/CoreLogic.pm]: sorry,but why do we need this?

**Comment by ya4ept** [comment]
@Mortimal see #2982

**Pull request closed without merging**